Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -325,17 +323,17 @@ - (SnapshotVersion)parseVersion:(NSNumber *_Nullable)version {
 - (DocumentViewChange)parseChange:(NSDictionary *)jsonDoc ofType:(DocumentViewChange::Type)type {
   NSNumber *version = jsonDoc[@"version"];
   NSDictionary *options = jsonDoc[@"options"];
-  DocumentState documentState = [options[@"hasLocalMutations"] isEqualToNumber:@YES]
-                                    ? DocumentState::kLocalMutations
-                                    : ([options[@"hasCommittedMutations"] isEqualToNumber:@YES]
-                                           ? DocumentState::kCommittedMutations
-                                           : DocumentState::kSynced);
 
   XCTAssert([jsonDoc[@"key"] isKindOfClass:[NSString class]]);
-  FieldValue data = [_reader parsedQueryValue:jsonDoc[@"value"]];
-  Document doc = Doc(util::MakeString((NSString *)jsonDoc[@"key"]), version.longLongValue, data,
-                     documentState);
-  return DocumentViewChange{doc, type};
+  google_firestore_v1_Value data = [_reader parsedQueryValue:jsonDoc[@"value"]];
+  MutableDocument doc =
+      Doc(util::MakeString((NSString *)jsonDoc[@"key"]), version.longLongValue, data);
+  if ([options[@"hasLocalMutations"] isEqualToNumber:@YES]) {
```

Comment:
Question: is it necessary to mix numbers and booleans like this? Is there some `isEqualToBoolean` function?

Reply:
Updated to use `boolValue`.